solidity/contracts/system/VendingMachine.sol

```diff
@@ -20,9 +20,46 @@ contract VendingMachine is TBTCSystemAuthority{
     TBTCDepositToken tbtcDepositToken;
     FeeRebateToken feeRebateToken;
 
+    uint256 createdAt;
+
     constructor(address _systemAddress)
         TBTCSystemAuthority(_systemAddress)
-    public {}
+    public {
+        createdAt = block.timestamp;
+    }
+
+    /// @notice return the outstanding minted TBTC supply
+    function getMintedSupply() public view returns (uint256) {
+        return tbtcToken.totalSupply();
+    }
+
+    /// @notice get the maximum TBTC token supply in BTC * 10 ** 18 based on the
+    /// age of the contract deployment
```

Comment:
Still some room here to elaborate in English IMO.

Reply:
Do you want a translation of the method into English? Fine with me, just unclear what level we're looking for

Reply:
Eh, I think there's room to include the supply schedule itself in brief English, yeah. Beautiful revision IMO :)